torq.q

```diff
@@ -142,8 +142,13 @@ getapplication:{$[0 = count a:@[{read0 x};hsym last getconfigfile"application.tx
 
 // Logging functions live in here
 
+colours:(`ERROR`ERR`WRN`WARN`INF!("\033[1;31m";"\033[1;31m";"\033[0;33m";"\033[0;33m";"\033[0m"));
 // Format a log message
-format:{[loglevel;proctype;proc;id;message] "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;message)}
+if[.z.o like "w*";
+ format:{[loglevel;proctype;proc;id;message] "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;message)}];
+
+if[not .z.o like "w*";
+ format:{[loglevel;proctype;proc;id;message]((colours loglevel), "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;message)),"\033[0m"}];
 
```

Comment:
Alternatively you could define a vanilla format function and then just wrap it if not on windows:
`format:{[func;loglevel;proctype;proc;id;message] colours[loglevel],func[loglevel;proctype;proc;id;message],"\033[0m"}[format]`

Also, what if someone doesn't want colour? Can there be some option to turn it off even if its in `torq.q`?